src/App.php

```diff
@@ -225,90 +215,29 @@ public function redirect(string $route, string $target, int $code = Response::ST
         $this->any($route, new RedirectHandler($target, $code));
     }
 
+    /**
+     * Runs the app to handle HTTP requests according to any registered routes and middleware.
+     *
+     * This is where the magic happens: When executed on the command line (CLI),
+     * this will run the powerful reactive request handler built on top of
+     * ReactPHP. This works by running the efficient built-in HTTP web server to
+     * handle incoming HTTP requests through ReactPHP's HTTP and socket server.
+     * This async execution mode is usually recommended as it can efficiently
+     * process a large number of concurrent connections and process multiple
+     * incoming requests simultanously. The long-running server process will
```

Comment:
Typo

```suggestion
     * process a large number of concurrent connections and process multiple
     * incoming requests simultaneously. The long-running server process will
```

Reply:
Thank you, updated as suggested! :shipit: